controllers/user.js

```diff
@@ -230,6 +229,18 @@ async function getDashboard(req, res) {
   const allBreaches = req.app.locals.breaches;
   const { verifiedEmails, unverifiedEmails } = await getAllEmailsAndBreaches(user, allBreaches);
 
+  let experimentBranch = null;
+  let isUserInExperiment = null;
+  let experimentBranchB = null;
```

Comment:
`let` = code sniff. Can we wrap these up in a single object and do something more like:
```js
const experimentFlags = _getExperimentFlags(req);
...
res.render("dashboards", {
...
  experimentFlags,
});
```